monkey/monkey_island/cc/services/config.py

```diff
@@ -246,8 +246,8 @@
                             "default": [
                             ],
                             "description":
-                                "List of IPs/subnets the monkey should scan."
-                                " Examples: \"192.168.0.1\", \"192.168.0.5-192.168.0.20\", \"192.168.0.5/24\""
+                                "List of IPs/subnets/hostnames the monkey should scan."
+                                " Examples: \"192.168.0.1\", \"guardicore.com\", \"192.168.0.5-192.168.0.20\", \"192.168.0.5/24\""
```

Comment:
Not guardicore.com, provide an example server, not an apex domain.
Preferably a dummy example like mj43printer